### PR TITLE
Update deployment status

### PR DESCRIPTION
**Pull request by bdunne**
TPINVTRY-532

### CODE REVIEW

File: lib/topological_inventory/orchestrator/worker.rb [hunk 1]
```diff
@@ -100,15 +100,16 @@ def collectors_from_sources_api
...
           key = digest(value)
           hash[key] = value
         end
         hash
       end
 
-      def sources_api_url_for(path)
-        File.join(sources_api, path)
+      def sources_api_url_for(*path)
+        File.join(sources_api, *path)
```
Comment (agrare) [inline lib/topological_inventory/orchestrator/worker.rb]: This was a nice change :+1:

File: lib/topological_inventory/orchestrator/worker.rb [hunk 2]
```diff
@@ -100,15 +100,16 @@ def collectors_from_sources_api
               "password" => authentication["password"],
               "username" => authentication["username"],
             },
+            "tenant"          => tenant,
```
Comment (slemrmartin) [inline lib/topological_inventory/orchestrator/worker.rb]: @bdunne Why do you add tenant into digest? If it's possible to change tenant for source, then it is still the same provider with the same endpoint and credentials.
Changing of digest means that this is new source from orchestrator's/openshift's point of view => which means kill and redeploy. 
And this is not necessary.

Comment (bdunne) [reply lib/topological_inventory/orchestrator/worker.rb]: I need it to update the source record [here](https://github.com/ManageIQ/topological_inventory-orchestrator/pull/34/files#diff-4743ee12ee7e621468f2e6590de994efR218)

Comment (slemrmartin) [reply lib/topological_inventory/orchestrator/worker.rb]: ah, ok, so you don't need it for digest but for rest api call. 
I won't keep it in #31, tenant will be saved in `Source` instance

**Pull request merged**